packages/cloud-cognitive/src/components/PageHeader/PageHeader.js

```diff
@@ -630,12 +661,18 @@ export let PageHeader = React.forwardRef(
                       className={cx(`${blockClass}__title`, {
                         [`${blockClass}__title--fades`]: hasBreadcrumbRow,
                       })}>
-                      {TitleIcon ? (
+                      {TitleIcon && !titleLoading ? (
                         <TitleIcon className={`${blockClass}__title-icon`} />
-                      ) : (
-                        ''
-                      )}
-                      <span title={title}>{title}</span>
+                      ) : null}
+                      <span title={titleText}>
```

Comment:
It seems that in most other places, you check that you either have the `titleText` before rendering it _or_ that  `titleLoading` is false before rendering `titleText`. Should that be the same in this spot @lee-chase?

Reply:
Maybe something like this

```jsx
<span title={!titleLoading ? titleText : null}>
```

Reply:
Agreed.